### DISCUSSION TITLE
How to use lens from a specific URL

### DISCUSSION THREAD

**Discussion by rtnj1994** [ptrumpis/snap-camera-server#139] [Q&A] [answered]
Hi, I'm Brazilian so I don't know if I did it right, I did the step by step to create the server, from what I understand by doing this step by step, I would have access to all the lenses available here: https://lens.snapchat.com /

I did the step by step, opened snapcamera, but I didn't find the lens I wanted, it's here: https://www.snapchat.com/lens/63ab23c56f1a450696baa96a8c5cd118?type=SNAPCODE&metadata=01

HOW CAN I GET ACCESS TO THIS LENS THROUGH SNAPCAMERA? please someone help me.

**Comment by TheVic489** [comment, marked as answer]
put that url in the search bar of snap camera and click it 
if it don't works, use the snap camera tools, there is a video tutorial of that 

VIDEO TUTORIAL
https://youtu.be/Pp6ljLzRjnU?si=9uk7DCv_GbI9r10W
